app/cmd/cli/query.go

```diff
@@ -94,6 +96,14 @@ var queryHeight = &cobra.Command{
 		fmt.Printf("Block Height: %d\n", res)
 	},
 }
+var version = &cobra.Command{
```

Comment:
Command is under query not root 

```
oten91@OMP % ./pocket_core version
Error: unknown command "version" for "pocket"
Run 'pocket --help' for usage.
unknown command "version" for "pocket"
oten91@OMP pocket_core % ./pocket_core query version
Version: RC-0.2.0
oten91@OMP pocket_core % 
```

is this intended?

Reply:
It is intended, however we can move this to root if everyone agrees.